certified-connectors/The Bot Platform Connector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,304 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "The Bot Platform",
+    "description": "Connect The Bot Platform's bots to any application, send messages to bot users, set attribute values using your existing bots at The Bot Platform",
+    "version": "1.0",
+    "contact": {
+      "name": "The Bot Platform Support",
+      "url": "https://thebotplatform.com/https-thebotplatform-com-contact/",
+      "email": "hello@thebotplatform.com"
+    }
+  },
+  "host": "api.thebotplatform.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://thebotplatform.com"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://thebotplatform.com/privacy-policy/"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Communication;Website"
+      }
+    ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1.0/activity/external": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@tbpsami it is not allowed to have a default response only. Please define a proper response. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses), 
Please fix all responses in the swagger file.

Reply:
@Amjed-Ayoub Is the schema needed for each responses?

Reply:
Ok done. I have defined responses everywhere on the swagger file.